aad/spring-cloud-azure-starter-active-directory/aad-resource-server/pom.xml

```diff
@@ -13,6 +13,7 @@
   <artifactId>spring-cloud-azure-starter-active-directory-resource-server</artifactId>
   <version>1.0.0</version>
   <packaging>jar</packaging>
+  <name>Spring Cloud Azure Starter Active Directory Resource Server</name>
```

Comment:
How about uniform the prefix? Use `Spring Cloud Azure Starter` in stead of `Azure Spring Boot Starter` in all places.

Reply:
Here is another option: Use format like 

```
${artifact-id}-sample-${feature-name}
```

Example:
```
spring-cloud-azure-starter-active-directory-sample-resource-server
```